packages/cozy-search/src/components/Search/Icons/EmailIcon.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+// TODO: should be moved in cozy-ui
```

Comment:
Why not add this icon now in cozy-ui?

Reply:
Yes, let's do that (but not blocking for this PR)

Reply:
It is usually 10 minutes to add the icon, and then you just need to update cozy-ui in your lib and use the new icon from cozy-ui.

Reply:
@AlaeMghirbi Since the icon exists in cozy-ui, could you:
- Remove the EmailIcon file
- Replace `import EmailIcon from '../../Search/Icons/EmailIcon'` by `import MailIcon from 'cozy-ui/transpiled/react/Icons/Mail'` here https://github.com/cozy/cozy-libs/blob/0bad727a2dfe4c7fc62063492e88309e18df5996/packages/cozy-search/src/components/Conversations/Sources/EmailSourceItem.jsx#L10 

Then, all should be good!